sdk/src/message_interface/error.rs

```diff
@@ -0,0 +1,68 @@
+// Copyright 2023 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+use packable::error::UnexpectedEOF;
+use serde::{Serialize, Serializer};
+
+pub use super::{message::AccountMethod, response::Response};
+use crate::serde::ser::SerializeMap;
+
+// TODO: SDK Error instead?
+/// Error type of the message interface.
+#[derive(Debug, thiserror::Error)]
+#[allow(clippy::large_enum_variant)]
```

Comment:
Can we fix this? Should we open an issue for this?

Reply:
large_enum_variant can only be fixed be merging variants? Don't see where that would make sense

Reply:
no, it means that some variant is too big and probably should be boxed.

Reply:
the thing is that Rust always has to create as much space such that its biggest variant can fit in which could be inefficient, that's why for very big variants it's better to just use a pointer.

Reply:
Ah yeah I thought it was about many variants like the `too_many_arguments` lint
I think boxing was annoying to do in some places, but not sure exactly, we have this on many Error enums

Reply:
yeah, let's make this part of another PR :+1: I'll open ... <drum-roll> an issue!

Reply:
opened #199